app/views/static_pages/team.html.erb

```diff
@@ -326,19 +326,14 @@
                 <%= link_to image_tag("partners/CODE.png", class: "logo"),
                 "http://codedocumentary.com/", target: "_blank" %>
             </div>
-             <div class="partner col-sm-4">
+            <div class="partner col-sm-4">
                 <%= link_to image_tag("partners/JETBRAINS.png", class: "logo"),
                 "https://www.jetbrains.com/", target: "_blank" %>
             </div>
             <div class="partner col-sm-4">
                 <%= link_to image_tag("partners/devbootcamp.png", class: "logo"),
                 "https://devbootcamp.com/", target: "_blank" %>
             </div>
-         
-          </div><!-- /row -->
-          <div class="row">
-       
-           
         </div><!-- /row -->
```

Comment:
I saw you added the fourth logo into the first row. We are using bootstrap row and col-sm-4 elements here, to build a grid with 3 logos per row. col-sm-4 designates the size of the column, and the number should always add up to 12 per row. You can read up more about that at [W3schools](http://www.w3schools.com/bootstrap/bootstrap_grid_system.asp). 
I'd say just add the second row back and put the devbootcamp logo in that row. Thank you!